contracts/router/interfaces/algebra/IAlgebraStaticQuoter.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity 0.8.17;
+
+import {QuoteExactInputSingleParams} from "./AlgebraStructs.sol";
+
+interface IAlgebraStaticQuoter {
+    function quoteExactInputSingle(QuoteExactInputSingleParams memory params) external view returns (uint256 amountOut);
```

Comment:
`quoteExactInputSingle` seems to return `(uint256 amountOut, uint16 fee)` from [this quoter contract](https://basescan.org/address/0x9c8F11C113F1fE7959C838a5F6b43e5E6223320c#code)?

Reply:
Looks like a different quoter than the one you're specifying tho. Plucked it from algebra docs

Reply:
We're using the read-only third-part quoter there, similar to UniV3 module. Algebra quoter doesn't expose a static method for quoting.

https://github.com/eden-network/uniswap-v3-static-quoter/blob/master/contracts/AlgebraQuoter/AlgebraStaticQuoter.sol